src/lib.rs

```diff
@@ -27,16 +27,15 @@
 //! | `aws_lc_rs` | Enable use of the aws-lc-rs crate for cryptography. |
 
 #![no_std]
-#![warn(elided_lifetimes_in_paths, unreachable_pub)]
+#![warn(elided_lifetimes_in_paths, unreachable_pub, clippy::use_self)]
 #![deny(missing_docs, clippy::as_conversions)]
 #![allow(
     clippy::len_without_is_empty,
     clippy::new_without_default,
     clippy::single_match,
     clippy::single_match_else,
     clippy::type_complexity,
-    clippy::upper_case_acronyms,
-    clippy::use_self
+    clippy::upper_case_acronyms
```

Comment:
nit: keep trailing comma?

Reply:
err, assuming that's allowed by `allow()`!

Reply:
`cargo fmt` erases it :-(

Reply:
I looked into that, unfortunately rustfmt has a bug -- there's a PR but it's languishing.

Reply:
Bummer :-(